README.md

```diff
@@ -105,3 +105,11 @@ List of 9
 ```
 
 *Note:* The sources (tank, separators, wellheads, etc.) will vary based on site-level details unique to users' contexts. But at a minimum, users can see that times (vector), sensor observations (data frame), wind data (vectors: direction/`WD` and speed/`WS`), and at least one source (data frame) are required to be in the list. 
```

Comment:
Do you think this note is still needed given your additions? If not, feel free to adjust the commit to cut this part in favor of your additions. Otherwise, you can leave as is; your call

Reply:
Nope I'll take it out.